src/tasks/test-deployment.ts

```diff
@@ -83,11 +86,11 @@ interface CleanArgs {
   weth: Token;
   usdcPerWeth: BigNumber;
   chainId: SupportedChainId;
-  gnosisDaoAddress?: string;
-  cowDaoAddress?: string;
-  communityFundsTargetAddress?: string;
-  investorFundsTargetAddress?: string;
-  teamControllerAddress?: string;
+  gnosisDaoAddress: string | undefined;
+  cowDaoAddress: string | undefined;
+  communityFundsTargetAddress: string | undefined;
+  investorFundsTargetAddress: string | undefined;
+  teamControllerAddress: string | undefined;
```

Comment:
why these changes?

Reply:
I was adding `claimCsv` to the same interface, and that field has the same restrictions as these. I realized that these fields were never really optional and were always there, but just be set to `undefined` sometimes. So I fixed all the types here since it's a very minor change.

Reply:
okok, i thought it was equivalent and preferable the `?` form, but i guess u want to signal its presence even if they are undefined. NVM, just curious